eagle-core/eagle-policy/eagle-policy-base/src/main/java/org/apache/eagle/policy/siddhi/SiddhiPolicyEvaluator.java

```diff
@@ -130,6 +133,7 @@ private SiddhiRuntime createSiddhiRuntime(SiddhiPolicyDefinition policyDef) {
 
         try {
             executionPlanRuntime = siddhiManager.createExecutionPlanRuntime(executionPlan);
+            executionPlanRuntime.handleExceptionWith(new SiddhiPolicyExceptionHandler());
```

Comment:
My patch is to extend capability of customizing exception handler for siddhi while it uses FatalExceptionHandler by default, so here we should override our own exception handler `SiddhiPolicyExceptionHandler` and explicitly specify by `handleExceptionWith`.